components/dropbox/actions/list-file-folders-in-a-folder/list-file-folders-in-a-folder.mjs

```diff
@@ -0,0 +1,113 @@
+import dropbox from "../../dropbox.app.mjs";
+import isNil from "lodash/isNil.js";
+import get from "lodash/get.js";
+import consts from "../../consts.mjs";
+
+export default {
+  name: "List All Files/Subfolders in a Folder",
+  description: "Searches for files and folders by name [See the docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesSearchV2__anchor)",
+  key: "dropbox-list-all-files-subfolders-in-a-folder",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    query: {
+      propDefinition: [
+        dropbox,
+        "query",
+      ],
+    },
```

Comment:
This action is searching for files/folders by name, but it's meant to simply list all files/folders at a path. Since `query` is required, I'm not seeing a way to do that with this action. I wonder if this endpoint would make more sense? https://www.dropbox.com/developers/documentation/http/documentation#files-list_folder